examples/public/js/web-eid.js

```diff
@@ -1,7 +1,7 @@
 /**
  * MIT License
  *
- * Copyright (c) 2020-2021 Estonian Information System Authority
```

Comment:
Maybe we need revert this and update web-eid.js from source in separate PR

Reply:
Fixed